### DISCUSSION TITLE
How to reset selected value?

### DISCUSSION THREAD

**Discussion by BigDaddyKahuna** [fraserxu/react-dropdown#210] [Q&A]
It appears that once you select an option from the dropdown, that value remains selected.   I would like to reset the value to the default value after capturing the user's selection.   Is this possible?

**Comment by boffobos** [comment]
Hello!
I found some approach to reset selected value to default. There is a attribute called 'value'. It is possible to set option to this attribute via useEffect hook. Option should be like this: 
```ts
import Option from 'react-dropdown';

export const Menu = ( {options}: Option) ReactElement => {
const [ defaultPlaceholder, setDefaultPlaceholder ] = useState<Option>();
const defVal = {
label: 'default placeholder',
value: 'default value'
};
useEffect(() => {
  if(someCond) setDefaultPlaceholder(defVal);
})
return <Dropdown value={defaultPlaceholder} />
